sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -3,12 +3,20 @@
 ## 4.0.0-beta.5 (Unreleased)
 
 ### Features Added
+- Added support for address type field value.
```

Comment:
I think you're missing mentioning DocumentPageKind here?

Reply:
I will add the missing changelog and readme updates in the following PR.